test/studies/lcals/RunVectorizeOnlyRawLoops.chpl

```diff
@@ -480,11 +480,11 @@ module RunVectorizeOnlyRawLoops {
             loop_data.RealArray_scalars[0] = (val + 0.00123) / (val - 0.00123);
             loopFinalize(iloop, stat, ilength);
           }
-          when LoopKernelID.PIC_2D {
-            halt("multidim cases not implemented ", iloop:LoopKernelID);
-            loopInit(iloop, stat);
-            loopFinalize(iloop, stat, ilength);
-          }
+          //when LoopKernelID.PIC_2D {
```

Comment:
I'm planning on implementing the parallel and vectorizeOnly variants of PIC_2D in pretty short order.  I commented them out for now so the serial variant can run and the parallel/vectorizeOnly variants of other kernels can run without the "halt" hitting here.  So I prefer to leave this commented out bit until it's implemented soon.